docs/implementation.html

```diff
@@ -103,67 +103,8 @@ <h5 class="anchor-heading"><a id="recordheader" class="anchor-link"></a><a href=
 
     <h4 class="anchor-heading"><a id="messageset" class="anchor-link"></a><a href="#messageset">5.3.3 Old Message Format</a></h4>
     <p>
-        Prior to Kafka 0.11, messages were transferred and stored in <i>message sets</i>. In a message set, each message has its own metadata. Note that although message sets are represented as an array,
-        they are not preceded by an int32 array size like other array elements in the protocol.
+        Prior to Kafka 0.11, messages were transferred and stored in <i>message sets</i>. See <a href="https://kafka.apache.org/39/documentation/#messageset">Old Message Format</a> for more details.
```

Comment:
Should we document that users cannot write to v0/v1, but reading is permitted?

Reply:
I debated that, but it seems like a very specific edge case and probably doesn't matter enough to add here. The only way this can happen is if you upgrade from an older version where the IBP was older than 3.0 and message format was set to v0/v1 and the topic was compacted without a deletion policy.

Reply:
Yes, that is an edge case. Also, when I googled the keywords "kafka 4.x message v0," KIP-724 appeared in the results. I believe users can refer to KIP-724 for the answer. Therefore, I agree that we don't need to add such documentation.

Reply:
Right, the KIP covers the edge cases in detail and is linked from the upgrade notes.